2Dgame/Assets/Scripts/Platform Scripts/BlinkingPlatformScript.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BlinkingPlatformScript : OnJumpScript {
+
+    private GameControllerScript gcs;
+    private PlatformSpawnerScript spawnerScript;
+
+    private new void Awake()
+    {
+        GameObject gameController = GameObject.Find("GameController");
+        gcs = gameController.GetComponent<GameControllerScript>();
+        spawnerScript = gameController.GetComponentInChildren<PlatformSpawnerScript>();
+        base.Awake();
+    }
+
+
+    /**
+     * Makes the GameObject blink for a bit and the disappear
+     * 
+     * parameter renderer: The renderer of the GameObject
+     */
+    IEnumerator Disappear(Renderer renderer)
+    {
+        yield return new WaitForSeconds(Random.Range(0f, 0.2f));
+        bool boolean = false;
+        for (int i = 0; i < 7; i++)
+        {
+            if (renderer != null)
+            {
+                renderer.enabled = boolean;
+            }
+
+            yield return new WaitForSeconds(0.1f);
+            boolean = !boolean;
+        }
+    }
+
+    /**
+     * Blinks away the next three blinkers
+     */
+    protected override void OnJump()
+    {
+        GameObject[] blinkers = GameObject.FindGameObjectsWithTag("Blinker");
+        StartCoroutine(gcs.PauseForSeconds(2));
```

Comment:
Maybe this should be slightly shorter? I feel like it's fine for the flips but for the blinkers I think it should be shorter.